buttons.py

```diff
@@ -4,7 +4,8 @@
 class Button:
     _instances=[]
     
-    def __init__(self,name,(x1,y1),centre=True): # eg ('plus',(30,40))
+    def __init__(self,name, coordinates,centre=True): # eg ('plus',(30,40))

```

Comment:
Usually we would have spaces after commas in an argument list, but on the other hand if that would require a huge change for consistency, then best not to.